src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -424,6 +426,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                        help='Bypass traffic for space-separated uses.')
             c.argument('default_action', arg_type=get_enum_type(t_default_action),
                        help='Default action to apply when no rule matches.')
+            c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
+            c.argument('vnet_name', help='Name of a virtual network.', validator=validate_subnet)
+            c.argument('action', help='The action of virtual network rule.')
```

Comment:
These arguments are defined for both `storage account create` and `storage account update`. But I didn't find them in `update_storage_account` within `custom.py`. Why?

Reply:
we do not support it in update command as you can see in description

Reply:
Why not move the definition to `storage account create` specific part then?